datamonster_api/lib/datasource.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, _id, name, category, uri, dm):
         self.uri = uri
         self.dm = dm
 
+    def __hash__(self):
+        return hash(self.id)
```

Comment:
why hash(self.id) instead of just self.id?

Reply:
from the docs " __hash__() should return an integer."

Reply:
i see. i suppose `self.id` here is a uuid?

Reply:
yes